app/uk/gov/hmrc/economiccrimelevyregistration/views/templates/Layout.scala.html

```diff
@@ -31,8 +33,29 @@
         autocompleteJavascript: HmrcAccessibleAutocompleteJavascript
 )
 
-@(pageTitle: String, showBackLink: Boolean = true, timeout: Boolean = true, showSignOut: Boolean = true, isWelshTranslationAvailable: Option[Boolean] = None, autoCompleteEnabled: Boolean = false)(contentBlock: Html)(implicit request: Request[_], messages: Messages)
+@(pageTitle: String,
+        showBackLink: Boolean = true,
+        timeout: Boolean = true,
+        showSignOut: Boolean = true,
+        isWelshTranslationAvailable: Option[Boolean] = None,
+        autoCompleteEnabled: Boolean = false,
+        registrationTypeForTopBanner: Option[RegistrationType] = Some(Initial),
+        eclReferenceForStartLink: Option[String] = None
+)(contentBlock: Html)(implicit request: Request[_], messages: Messages)
+
+@topPageTitle = @{
+    registrationTypeForTopBanner match {
+        case Some(Amendment) => Some(messages("start.amend.title"))
+        case _ => None
```

Comment:
What shows for the service name when this is None?

Reply:
It will show standard title, defined with key `service.name ` which in our case is _Register for the Economic Crime Levy_